azure_monitor/azfunc_sample/README.md

```diff
@@ -1,90 +1,78 @@
 ---
-page_type: sample
-languages:
-- python
-products:
-- azure
-description: "This sample contains a simple Azure Function to show how you can send correlated application log information as well as information of external dependencies to Azure Monitor (Application Insights)."
-urlFragment: azure-monitor-opencensus-python
+title: 'Sample: Configuring Function Apps with Python and OpenCensus'
+description: Create Azure Functions using Python and OpenCensus.
+author: givenscj
+ms.author: givenscj
+ms.devlang: python
+ms.topic: sample
+ms.date: 07/27/2022
+ms.custom: []
 ---
 
-# Correlated End-To-End Azure Function (Python) Instrumentation with Opencensus 
+# Sample: Configuring Function Apps with Python and OpenCensus
 
-## Overview
+This is an example of how to add the various settings Python-based Azure Function applications to send dependencies calls to Azure Monitor.  You will run the samples both locally and deploy the Azure Functions to an Azure App Service resource you deployed as part of the setup script.
 
-Azure Function (Python) already supports instrumentation using Opencensus. The following sample utilizes the OpenCensus Functions extension and enables additional common telemetry scenarios used by Azure Monitor and Functions users. Such scenarios include:
-- End-to-end view of a single function invocation (in Azure Monitor Application Insights)
-- Create an application topology view (application dependencies to support distributed tracing) using Azure Monitor Application Insight's Application Map (especially useful when working with distributed, scalable applications). 
+## 1 - Environment Setup
 
-### Example of a Correlated End-to-End Azure Function Invocation
+Follow the steps in the [setup documentation](/azure_monitor/readme.md).
 
-End-to-end transaction:
+## 2 - Utilize Function Apps with OpenCensus
 
-![AI Correlated end2end function invocation](./docs/assets/01-ai-correlated-invocation.PNG)
+- Start the Azure Storage Emulator.
```

Comment:
What is `Azure Storage Emulator` and how do you start it?

Reply:
Oh I see that it is linked [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator). Maybe we can mark this as `optional` since it is not really needed to get the sample running.

Reply:
One of the steps in the functions sample is to run the sample locally, and you would need it for that.  I added some extra text that it is for the function app sample.